.github/workflows/update-website.yml

```diff
@@ -0,0 +1,25 @@
+name: Update website
+
+on:
+  push:
+    branches:
+      - master
+    paths:
+      - "integrations/integrations.json"
+
+jobs:
+  trigger:
```

Comment:
This job should have a conditional so that it only runs on in the `netdata/netdata` repository. Otherwise it will trigger whenever anybody syncs their fork of the repo, and then pester them about failed jobs that are not at all relevant to them.

Reply:
Added that